### DISCUSSION TITLE
Litex-cnc on standalone fpga chip?

### DISCUSSION THREAD

**Discussion by spacehitchhiker42** [Peter-van-Tol/LiteX-CNC#102] [Q&A] [answered]
Would it be possible to use this firmware on a custom board? Basically it would just be an ecp5 fpga and an rmii phy. I know that the colorlight board has SDRAM, is that necessary, or can the litex SOC just use the internal memory of the fpga?

**Comment by Peter-van-Tol** [comment, marked as answer]
This is possible, because Litex-CNC does not use the DRAM.

Please share your design when you take this idea further. Also, when in need for custom components for your design, please let me know.
